iocore/utils/I_Machine.h

```diff
@@ -77,6 +77,11 @@ struct Machine {
   /// @return The global instance of this class.
   static self_type *instance();
   bool is_self(std::string_view name);
+  bool
+  is_self(char const *name)
+  {
+    return this->is_self(std::string_view(name));
+  }
```

Comment:
Probably best to keep the definition in `iocore/utils/Machine.cc` like the rest of the functions here.

Reply:
OK

Reply:
Not good for optimal inlining, but not called in any high-rep loop I don't think.